astro.config.mjs

```diff
@@ -182,24 +182,26 @@ export default defineConfig({
             collapsed: true,
             items: [
               { label: 'Overview', link: '/dropins/product-details/' },
-              { label: 'Installation', link: '/dropins/product-details/pdp-installation/' },
-              { label: 'Initialization', link: '/dropins/product-details/pdp-initialization/' },
-              { label: 'Styles', link: '/dropins/product-details/pdp-styles/' },
-              { label: 'Containers', link: '/dropins/product-details/pdp-containers/' },
-              { label: 'Slots', link: '/dropins/product-details/pdp-slots/' },
-              { label: 'Functions', link: '/dropins/product-details/pdp-functions/' }
+              { label: 'Installation', link: '/dropins/product-details/installation/' },
```

Comment:
You'll need to add redirects for all files that you moved.

Reply:
@jeff-matthews — Sorry. User error. I was working on my USF-921 branch and had made many of these changes on that branch. Should be good now.